assets/queries/k8s/hpa_targeted_deployments_with_configured_replica_count/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+CxPolicy [ result ] {  
+  metadata := input.document[i].metadata
+  replicasCheck := input.document[i].spec.replicas
+  kindCheck := input.document[i].kind
+  scaleKindCheck := input.document[_].spec.scaleTargetRef.kind
+ 
+  metadata.name == input.document[_].spec.scaleTargetRef.name
```

Comment:
Replace with:
```
scaleKindCheck := input.document[j].spec.scaleTargetRef.kind

metadata.name == input.document[j].spec.scaleTargetRef.name
```
Because the document where the query gets the `scaleTargetRef.kind` and `scaleTargetRef.name` must be the same.